Furball.Engine/Engine/Graphics/ContentManager.cs

```diff
@@ -103,7 +65,7 @@ public static Texture2D LoadTextureFromFile(string filename, ContentSource sourc
                         data = File.ReadAllBytes(path);
                 }
                 if ((int)source >= (int)ContentSource.Game && data.Length == 0) {
-                    path = Path.Combine(FurballGame.Instance.Content.RootDirectory, filename);
+                    path = Path.Combine(executablePath, filename);
```

Comment:
This needs to be fixed, it shouldnt be using the executable path

Reply:
what path should it use then

Reply:
> what path should it use then

A game customizable one, it defaulted to `Content` before so thats what it should be by default here

Reply:
i think ill add a contentpath string that defaults to that

Reply:
sounds good to me

Reply:
done